### PR TITLE
use _retryWithInterval in screenshotTools

### PR DESCRIPTION
**Pull request by RaananW**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Visualization tests for WebGL 1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15986/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15986/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15986/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/Misc/screenshotTools.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import type { Nullable } from "../types";
 import { ApplyPostProcess } from "./textureTools";
 
 import type { AbstractEngine } from "../Engines/abstractEngine";
+import { _retryWithInterval } from "../Materials/effect.functions";
```
Comment (sebavan) [inline packages/dev/core/src/Misc/screenshotTools.ts:15]: Should we move the function somewhere else ?

**Pull request closed without merging**